include/net/phy.h

```diff
@@ -0,0 +1,231 @@
+/**
+ * @file
+ *
+ * @brief Public APIs for Ethernet PHY drivers.
+ */
+
+/*
+ * Copyright (c) 2021 IP-Logix Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DRIVERS_PHY_H_
+#define ZEPHYR_INCLUDE_DRIVERS_PHY_H_
+
+/**
+ * @brief Ethernet PHY Interface
+ * @defgroup ethernet_phy Ethernet PHY Interface
+ * @ingroup networking
+ * @{
+ */
+#include <zephyr/types.h>
+#include <device.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief Ethernet link speeds. */
+enum phy_link_speed {
+	/** 10Base-T Half-Duplex */
+	LINK_HALF_10BASE_T		= BIT(0),
+	/** 10Base-T Full-Duplex */
+	LINK_FULL_10BASE_T		= BIT(1),
+	/** 100Base-T Half-Duplex */
+	LINK_HALF_100BASE_T		= BIT(2),
+	/** 100Base-T Full-Duplex */
+	LINK_FULL_100BASE_T		= BIT(3),
```

Comment:
better if you can keep same alignment

Reply:
Not sure what is being asked, specifically.